gnc/sub8_perception/nodes/marker_occ_grid.py

```diff
@@ -61,7 +61,10 @@ def add_circle(self, center, radius):
         '''
         center_offset = np.array(center) / self.meta_data.resolution - np.array([self.mid_x, self.mid_y])
 
-        radius = int(radius / self.meta_data.resolution)
+        try:
+            radius = int(radius / self.meta_data.resolution)
+        except:
```

Comment:
Damn, forgot what problem this had.
I think all the weird number errors are due to resetting the simulations and things not having enough time to settle in before other things get initialized. So probably not an "actual" error but rather a synthetic one.